source/wp-content/themes/wporg-developer-2023/patterns/reference-content.php

```diff
@@ -7,7 +7,7 @@
 
 ?>
 
-<!-- wp:group {"tagName":"main","style":{"spacing":{"padding":{"left":"var:preset|spacing|edge-space","right":"var:preset|spacing|edge-space","bottom":"var:preset|spacing|60"}}},"className":"alignfull","layout":{"type":"constrained","wideSize":"1280px","contentSize":"680px"}} -->
+<!-- wp:group {"tagName":"main","style":{"spacing":{"padding":{"left":"var:preset|spacing|edge-space","right":"var:preset|spacing|edge-space","bottom":"var:preset|spacing|60"}}},"className":"alignfull","layout":{"type":"constrained","justifyContent":"left"}} -->
```

Comment:
Having this section left-aligned looks kind of strange in practice, IMO. It is a "single column" but also kind of not, so I think it could be centered.

![Screen Shot 2024-02-15 at 12 56 40](https://github.com/WordPress/wporg-developer/assets/541093/7baa8b8f-c58c-4265-b2f0-554c0dfa9893)

Reply:
I'm drawn to keep this page untouched as the core problem here is content. Once we finish the improvements milestone we can open an issue and ping folks asking for ideas.

Reply:
I had the same thoughts. Thanks, I'll revert 👍

Reply:
Fixed in [31e083f](https://github.com/WordPress/wporg-developer/pull/499/commits/31e083fef93159b9b3b4e159799cf91e75a6f40e)